Formula/g/gettext.rb

```diff
@@ -27,6 +27,11 @@ class Gettext < Formula
   end
 
   def install
+    # macOS iconv implementation is slightly broken since Sonoma.
```

Comment:
macOS 14 iconv is not "slightly broken". It is completely buggy. Totally unusable for any serious purpose. See https://lists.gnu.org/archive/html/bug-gnulib/2024-05/msg00375.html for details.